website/learn/glossary/glossary/index.md

```diff
@@ -0,0 +1,81 @@
+---
+layout: learn
+title: Pactus Glossary
+sidebar: Pactus Glossary
+---
+
+<!-- TODO: find a name for pactus merkel root model and add it here (?) -->
+
+# A
```

Comment:
```suggestion
## A
```

and same for others

Reply:
@b00f we will get linting issues with that. we can increase the header level by only 1 for each header!